ouroboros-network/src/Ouroboros/Network/NodeToClient/Version.hs

```diff
@@ -41,6 +41,8 @@ data NodeToClientVersion
     -- ^ 'LocalStateQuery' protocol codec change, allows to acquire tip point.
     | NodeToClientV_9
     -- ^ enabled @CardanoNodeToClientVersion7@, i.e., Alonzo
+    | NodeToClientV_10
+    -- ^ enable @QueryV_1@, i.e. top level queries.
```

Comment:
I know this PR is currently supposed to be minimal, and so it doesn't actually add a top-level query. But I think we should prefer actually adding a query before `V_10` gets released. It'd be confusing to genuinely allocate a version that provides no actual downstream enrichment.

The actionable thing I'm suggesting probably requires some cooperative planning, eg involving the following events a) merging this PR b) merging the next PR that actually justifies a version bump c) updating downstream libraries' dependencies to involve those PRs we will have merged.

I had originally thought that (c) would be about making a proper release, but that's a little narrower than truth, unless we're willing to get a little clever here https://github.com/input-output-hk/ouroboros-network/blob/eb02c2d627b728a48449d314e56c347f1798503b/ouroboros-consensus/src/Ouroboros/Consensus/Node.hs#L717-L718

(This is probably hard to follow; consider it a mixture of pointing out a concern and making a note to my future self.)